File: Java/src/main/java/org/sisuns/condicionales/basico/Ejercicio_05.java
package org.sisuns.condicionales.basico;

public class Ejercicio_05 {
    public static void main(String[] args) {
        // Peso y altura predeterminados para el ejemplo
        double peso = 70.0;
        double altura = 1.75;

        // Calcular el Índice de Masa Corporal (IMC)
        double imc = peso / (altura * altura);

        // Determinar la categoría del IMC
        if (imc < 18.5) {
            System.out.println("Bajo de peso");
        } else if (imc >= 18.5 && imc < 25) {
            System.out.println("Peso normal");
        } else if (imc >= 25 && imc < 30) {
            System.out.println("Sobrepeso");
        } else {
            System.out.println("Obesidad");
        }
    }
}

File: Python/Condicional/Basico/Ejercicio_05.py
# Solicitar al usuario ingresar su peso y altura
peso = float(input("Ingrese su peso (kg): "))
altura = float(input("Ingrese su altura (m): "))

# Calcular el Índice de Masa Corporal (IMC)
imc = peso / (altura * altura)

# Determinar la categoría del IMC
if imc < 18.5:
    print("Bajo de peso")
elif 18.5 <= imc < 25:
    print("Peso normal")
elif 25 <= imc < 30:
    print("Sobrepeso")
else:
    print("Obesidad")
